curl/types.h and curl/easy.h includes are deprecated
See also:
http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=636558
Forwarded to Upstream via e-mail.
--- a/generic/tclcurl.h
+++ b/generic/tclcurl.h
@@ -16,8 +16,6 @@
 #endif
 
 #include <curl/curl.h>
-#include <curl/types.h>
-#include <curl/easy.h>
 #include <tcl.h>
 #include <tclDecls.h>
 #include <stdio.h>
